### ISSUE TITLE
Current PHAR distribution errors on PHP 8.1

### ISSUE THREAD

**Issue by mariusklocke** [Smile-SA/gdpr-dump#66]
Current PHAR distribution of gdpr-dump (Version 2.3.0) errors on PHP 8.1. This **DOES NOT** happen when installing gdpr-dump from git (and do a fresh `composer install`). I guess it's related to the version of `symfony/console` shipped in the PHAR.

Edit: Okay, it's not the version of `symfony/console`. The error occurs due to the way gdpr-dump PHAR build works. The `Compiler` class uses `php_strip_whitespace` to strip all the comments from source code, which eliminates the `#[\ReturnTypeWillChange]` attribute.

### Preconditions

GdprDump Version: 2.3.0

PHP Version: 8.1.2

Database Version: MariaDB 10.4

### Steps to reproduce

1. Just execute `php gdpr-dump.phar --help`

### Expected result
Help should be printed without errors

### Actual result
Fatal Error:
```
Fatal error: During inheritance of IteratorAggregate: Uncaught ErrorException: Return type of Symfony\Component\Console\Helper\HelperSet::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Helper/HelperSet.php:2
Stack trace:
#0 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Helper/HelperSet.php(2): Smile\GdprDump\AppKernel->Smile\GdprDump\{closure}(8192, 'Return type of ...', 'phar:///usr/loc...', 2)
#1 phar:///usr/local/bin/gdpr-dump/vendor/composer/ClassLoader.php(2): include('phar:///usr/loc...')
#2 phar:///usr/local/bin/gdpr-dump/vendor/composer/ClassLoader.php(2): Composer\Autoload\includeFile('phar:///usr/loc...')
#3 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Application.php(2): Composer\Autoload\ClassLoader->loadClass('Symfony\\Compone...')
#4 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Application.php(2): Symfony\Component\Console\Application->getDefaultHelperSet()
#5 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Command/Command.php(2): Symfony\Component\Console\Application->getHelperSet()
#6 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Application.php(2): Symfony\Component\Console\Command\Command->setApplication(Object(Smile\GdprDump\Console\Application))
#7 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Application.php(2): Symfony\Component\Console\Application->add(Object(Symfony\Component\Console\Command\HelpCommand))
#8 phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Application.php(2): Symfony\Component\Console\Application->init()
#9 phar:///usr/local/bin/gdpr-dump/src/AppKernel.php(2): Symfony\Component\Console\Application->add(Object(Smile\GdprDump\Console\Command\DumpCommand))
#10 phar:///usr/local/bin/gdpr-dump/bin/gdpr-dump(2): Smile\GdprDump\AppKernel->run()
#11 /usr/local/bin/gdpr-dump(6): require('phar:///usr/loc...')
#12 {main} in phar:///usr/local/bin/gdpr-dump/vendor/symfony/console/Helper/HelperSet.php on line 2
```

**Comment by guvra** [comment]
Hi,

Thanks for the report.
This is due to php_strip_whitespace removing PHP annotations when executed on PHP < 8.0.

I pushed a commit that replaces php_strip_whitespace with a custom function here: #67

**Comment by guvra** [comment]
The fix is published in version 2.3.1 :+1:

**Comment by mariusklocke** [comment]
Thank you for the quick response. Works perfectly with `2.3.1`. `gdpr-dump` is a really awesome tool :)
